global.json

```diff
@@ -1,3 +1,7 @@
 {
-    "sources": [ "src" ]
-}
\ No newline at end of file
+    "sdk": {
+      "version": "2.1.513",
+      "rollForward": "patch",
```

Comment:
Do you know if v2.1.513 SDK is available in Azure DevOp? I tried to specify v2.2.401 previously, but it fails in the build lab with the error of not finding the specified version of SDK.

Reply:
Yes, VS 2017 agents have 2.1.513 installed. I used this version because this seems to be what you were *already* using, although it wasn't explicit. You were just using whatever was on the agent. But this was a problem because your build _breaks_ on newer .NET SDK versions, so as a step toward stable builds on all machines, and eventually moving to newer build agents, we need to 'pin' what you depend on so we can intentionally upgrade when we are ready with a build that is tested to work on that version.

But we should *not* depend on agents to have this installed. I would like to follow up after this PR with another PR that will install the .NET SDK specified in this file onto the agent, so that we no longer are at the mercy of the agent image to have the prereq that we need.

Reply:
As evidence that this works as-is, you can see my build based on this PR: https://dev.azure.com/devdiv/Personal/_build/results?buildId=3654797&view=results